packaging/deb/control/postinst

```diff
@@ -32,10 +32,14 @@ case "$1" in
   fi
 
   if [ ! -f $PROVISIONING_CFG_DIR ]; then
-    mkdir -p $PROVISIONING_CFG_DIR/dashboards $PROVISIONING_CFG_DIR/datasources
+    mkdir -p $PROVISIONING_CFG_DIR/dashboards $PROVISIONING_CFG_DIR/datasources $PROVISIONING_CFG_DIR/notifiers
     cp /usr/share/grafana/conf/provisioning/dashboards/sample.yaml $PROVISIONING_CFG_DIR/dashboards/sample.yaml
     cp /usr/share/grafana/conf/provisioning/datasources/sample.yaml $PROVISIONING_CFG_DIR/datasources/sample.yaml
-  fi 
+    cp /usr/share/grafana/conf/provisioning/notifiers/sample.yaml $PROVISIONING_CFG_DIR/notifiers/sample.yaml
+  elif [ ! -f $PROVISIONING_CFG_DIR/notifiers ]; then
```

Comment:
I would make this into a second separate if statement. Would simplify the code and reduce duplication.